<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>depositView</class>
 <widget class="QWidget" name="depositView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Срок размещения</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>430</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(248, 159, 91);
	border: none;
	color: rgb(0, 0, 0);
	border-radius: 15px;
}
QPushButton:pressed {
	background-color: rgb(153, 84, 255);
	border: none;
	color: rgb(0, 0, 0);
	border-radius: 15px;
}</string>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>120</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Процентная ставка</string>
   </property>
  </widget>
  <widget class="QPushButton" name="result_deposit_button">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>430</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(248, 159, 91);
	border: none;
	color: rgb(0, 0, 0);
	border-radius: 15px;
}
QPushButton:pressed {
	background-color: rgb(153, 84, 255);
	border: none;
	color: rgb(0, 0, 0);
	border-radius: 15px;
}</string>
   </property>
   <property name="text">
    <string>Посчитать</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>80</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>*В днях</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>40</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>у.е.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cумма вклада</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>290</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>у.е.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Налоговая ставка</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Периодичность выплат</string>
   </property>
  </widget>
  <widget class="QComboBox" name="period_type_payments">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>210</y>
     <width>191</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Ежедневно</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Раз в месяц</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Раз в квартал</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>В конце срока</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="capitalization_check">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>250</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Капитализация процентов</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Начисленные проценты</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Сумма налога</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Сумма на вкладе к концу срока</string>
   </property>
  </widget>
  <widget class="QLabel" name="profit_label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>290</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="tax_sum_label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>330</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="sum_deposit_label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>370</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>330</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>у.е.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>370</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>у.е.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>160</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="deposit_sum_line">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox {
	color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="maximum">
    <double>10000000000000000000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="period_line">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox {
	color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="maximum">
    <double>10000000000000000000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="percent_line">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>120</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox {
	color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="maximum">
    <double>10000000000000000000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="tax_line">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>160</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox {
	color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="maximum">
    <double>10000000000000000000000.000000000000000</double>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
